pkg/mcp/tools/types.go

```
@@ -212,6 +212,7 @@ type ProjectToolsetHandler interface {
 	// Project operations
 	ListProjects(ctx context.Context, namespaceName string, opts ListOpts) (any, error)
 	CreateProject(ctx context.Context, namespaceName string, req *gen.CreateProjectJSONRequestBody) (any, error)
+	UpdateProject(ctx context.Context, namespaceName, projectName, deploymentPipeline, displayName, description string) (any, error)
```

golangci-lint (2.12.1)

[error] The line is 129 characters long, which exceeds the maximum of 120 characters.

(lll)